src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -402,6 +402,7 @@ protected override int MakeSlot(BoundExpression node)
             switch (node.Kind)
             {
                 case BoundKind.ObjectCreationExpression:
+                case BoundKind.DynamicObjectCreationExpression:
```

Comment:
>case [](start = 16, length = 4)

Can we cover this change with test?

Reply:
This change is covered by `DynamicObjectCreation_03`. That test fails without this addition.

---
In reply to: [198627393](https://github.com/dotnet/roslyn/pull/28155#discussion_r198627393) [](ancestors = 198627393)